helm-sources/zookeeper-2/values/demo.yaml

```diff
@@ -0,0 +1,8 @@
+  staging.yaml#namespace
+NAMESPACE: staging
```

Comment:
Trengs helm for zookeeper2? Vi skulle vel bare starte med 1 instans?
If so, namespace=demo
volumename=fdk-dev-demo-zookeeper-2